src/DS/sds.c

```diff
@@ -848,17 +849,25 @@ static int ds_sds_compose_add_component_dependencies(xmlDocPtr doc, xmlNodePtr d
 
 				if (oscap_acquire_url_is_supported(href)) {
 					/* If the referenced component is remote one, do not include
-					 * it within the DataStream. Such component shall only be
-					 * downloaded once the scan is run. */
-					xmlFree(href);
-					continue;
+					* it within the DataStream. Such component shall only be
```

Comment:
Does this mean after this change it will get included?

Reply:
Not the contents, just a `component-ref` in the `<ds:checks>` that will reference the remote content, and a `<uri>` in each Benchmark component that uses the remote resource.